README.md

```diff
@@ -121,6 +121,9 @@ ncu "/^(?!react-).*$/" # windows
   - `0.1.0` → `0.2.1`
 - With `--target patch`, only update patch:
   - `0.1.0` → `0.1.2`
+- With `--target @next`, latest version with dist-tag=next is used
+  _NOTE: pre: true is required if you want to upgrade from a stable version to a dist-tag_
```

Comment:
Is this desired? It seems like the default behavior for non-latest dist-tags should assume `pre: true`, like the other non-latest targets. Tags are often used for prelease versions or alternative branches.

Reply:
Didn't give that too much thought. But you are right that "default" behavior for non-latest assumes pre: true. If there is some case where that is not true, one can always go pre: false.

Good find!